docs/software/media.md

```diff
@@ -556,13 +556,20 @@ Also Installs:
     - Enter the details of your path (See the image example below to add your USB drive music folder to Ampache).
     - Select the `add catalogue` button.
 
-    Remark: For Ampache to access custom directories, you must ensure read access to the folder, e.g. by:
+    !!! note "Check access rights on local custom directories"
+        For Ampache to access custom directories, you must ensure read access to the folder, e.g. by:
 
-    ```sh
-    chmod -R 775 /my/directory
-    ```
+        ```sh
+        chmod -R 775 /my/directory
+        ```
 
-    ![Ampache web interface screenshot with instructions how to add media](../assets/images/dietpi-software-media-ampacheaddcatalogue.png){: width="400" height="180" loading="lazy"}
+        ![Ampache web interface screenshot with instructions how to add media](../assets/images/dietpi-software-media-ampacheaddcatalogue.png){: width="400" height="180" loading="lazy"}
```

Comment:
This image is now shown within the note box, right? I suggest to move this above the box, since it is related to the instructions list above and skip the additional indentation so that it is shown below the whole list and not as part of the last list entry.